docs/source/usage/workflow-installation.rst

```diff
@@ -25,12 +25,11 @@ commands assume you are within it.
 
 First up, if you are at a specific site, you need to install the site specific
 configuration. This is done by running the ``install_restricted_files.sh``
-script, and pressing enter a couple times to accept the default options.
+script.
 
 .. code-block:: bash
 
   ./install_restricted_files.sh
```

Comment:
Adding parameters, and defaults
```suggestion
  ./install_restricted_files.sh [<branch-name>] [<repository-url>]
  
<repository-url> defaults to https://github.com/MetOffice/CSET-workflow
<branch-name> defaults to main. It cannot be omitted if <repository-url> is specified
```

Reply:
I'll checkout the getopts bit first.

Reply:
Yep. I made this suggestion before working on the getopts one; if getopts is adopted then this will need other modifications.